test/unit/controllers/adhoc-payment/get-index.controller.test.js

```diff
@@ -13,16 +13,12 @@ const paths = require('../../../../app/paths')
 const expect = chai.expect
 let product, response, service, $
 
-function asGBP (amountInPence) {
-  return currencyFormatter.format((amountInPence / 100).toFixed(2), { code: 'GBP' })
-}
-
 describe('adhoc payment index controller', function () {
   afterEach(() => {
     nock.cleanAll()
   })
 
-  describe('variable amount ADHOC payment with reference disabled', function () {
+  describe('ADHOC payment starting page', function () {
```

Comment:
Moving tests to new payment links version will remove coverage for V1 which is currently used. Probably fine as we will be enabling the functionality soon.

If it is not too much work we could have tests for both similar to https://github.com/alphagov/pay-products-ui/blob/master/app/controllers/pre-payment.controller.test.js#L45